flowlogs.tf

```diff
@@ -16,48 +16,28 @@ data "aws_iam_policy_document" "log_stream_action" {
   }
 }
 
-data "aws_iam_policy_document" "log_stream_trust" {
-  statement {
-    effect = "Allow"
-
-    actions = [
-      "sts:AssumeRole",
-    ]
-
-    principals {
-      type = "Service"
-
-      identifiers = [
-        "vpc-flow-logs.amazonaws.com",
-      ]
-    }
-  }
+module "flow_logs_role" {
+  count                 = var.flow_logs != null ? 1 : 0
+  source                = "github.com/schubergphilis/terraform-aws-mcaf-role?ref=v0.3.0"
+  name                  = "FlowLogsRole-${var.flow_logs.iam_role_name}"
```

Comment:
This is not required as it can be set on the var.flow_logs.iam_role_name and make the naming convention flexible.